huntflow_webhook_models/common_models/applicant_offer.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime
+from typing import Any, Dict, Optional
+
+from pydantic import BaseModel, Field
+
+from huntflow_webhook_models.consts import ApplicantOfferLogTypes
+
+
+class AccountApplicantOfferLog(BaseModel):
+    id: int = Field(..., description="Log ID", examples=[1])
+    type: ApplicantOfferLogTypes = Field(
+        ...,
+        description="Log type",
+        examples=[ApplicantOfferLogTypes.ADD],
+    )
+
+
+class ApplicantOffer(BaseModel):
+    id: int = Field(..., description="Offer ID", examples=[1])
+    account_applicant_offer_log: AccountApplicantOfferLog = Field(
+        ...,
+        description="Account applicant offer log",
+    )
+    applicant_offer_id: int = Field(..., description="Applicant offer ID", examples=[1])
```

Comment:
```suggestion
    applicant_offer_id: int = Field(..., description="Account applicant offer ID", examples=[1])
```

Reply:
I think it makes sense to add links to according API entities, like https://api.huntflow.ai/v2/docs#get-/accounts/-account_id-/offers/-offer_id-